docs/index.rst

```diff
@@ -360,6 +360,15 @@ you can go to :guilabel:`Settings > Record placements` to enable this feature.
 Page-specific keywords
 ----------------------
 
+.. tip::
+    EthicalAds uses a crawler (similar to a search engine)
+    to crawl our publishers' sites and figure out the appropriate keywords and topics for each
+    page where ads appear. Most publishers won't need to do anything for EthicalAds
+    to appropriately target the right advertisers to the right pages on publisher sites.
+    This API is mostly for SPAs or other non-traditional sites
+    where our crawler won't work.
```

Comment:
I think we still want tags and keywords for our publishers right? This doesn't quite capture the value of sending them to us, as I do think it's generally useful if the publisher has good data, but that's probably a stretch I guess.

Reply:
We definitely want tags and keywords, but unless they think there's a reason they can do better than the classifier or the client, I think we'd rather they didn't.

Reply:
Yea, at least with blogs and other types of content, there's specific keywords, but yea we don't want just a static set for the entire publisher or anything like that.